lib/network/modules/Clustering.js

```diff
@@ -850,15 +850,25 @@ class ClusterEngine {
   /**
    *
    * @param {Cluster.id} clusterId
+   * @param {Recursive} recursive
```

Comment:
Shouldn't this be `boolean` instead of `{Recursive}`? I would expect something like `@param {boolean} [recursive=false]`

Correct me if I'm wrong.